packages/nodes-base/nodes/ActiveCampaign/GenericFunctions.ts

```diff
@@ -63,6 +63,7 @@ export async function activeCampaignApiRequest(
 			return responseData[dataKey] as IDataObject;
 		}
 	} catch (error) {
+		if (error instanceof NodeApiError) throw error;
```

Comment:
there must be a better way to do this instead of having to repeat this so many times.
perhaps we can try moving this check inside the `NodeApiError` constructor.

Reply:
I did consider this, but what if we actually want to re-wrap error? and if we decide to do that for `NodeApiError` we probably want same policy for other error types
Despite a lot of files check this would be less invasive and better for performance, as it's simple check